db/schema.rb

```diff
@@ -29,6 +57,28 @@
     t.text "description", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "user_id", null: false
+  end
+
+  create_table "users", force: :cascade do |t|
+    t.string "username", null: false
+    t.string "first_name"
+    t.string "last_name"
+    t.integer "role", default: 2
```

Comment:
I guess you removed roles from the migration but it still exists in the schema file. Please run the migrations again and keep this up to date...